scripts/eksa_components_override.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -o pipefail
+set -x
+set -o nounset
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+BUNDLE_MANIFEST_URL="$1"
+CI="${CI:-false}"
+
+SED=sed
+if [ "$(uname -s)" = "Darwin" ]; then
+    SED=gsed
+fi
+
+if ! yq --version; then
+    echo "Cannot find yq executable. Build on Linux"
+    echo "or install yq on Mac (brew install yq)."
+    exit 1
+fi
+
+# In Prow, we can get the base commit and PR commit using these
+# environment variables and check their diff to see if files under
+# config folder changed.
+if [ "$CI" = "true" ]; then
```

Comment:
I might be wrong here, but this check would work for presubmits but wouldn't the post submits fail?
I mean until the new release-bundle is generated with the new CRDs.

Reply:
Technically it would, but I've not seen that happen before so left it out. But I can extend this to Codebuild for completeness